lib/model/rwfolder.go

```diff
@@ -1645,8 +1633,8 @@ func (f *sendReceiveFolder) newError(path string, err error) {
 	if _, ok := f.errors[path]; ok {
 		return
 	}
-
-	f.errors[path] = err.Error()
+	l.Infof("Puller (folder %q, file %q): %s: %v", f.folderID, path, context, err)
```

Comment:
This should be debug, otherwise it will be logged repeatedly on puller iterations.

Reply:
It shouldn't I thinkn as we check the map before?

Reply:
Ah sorry, sloppy code reading.
Then this loop I recently added isn't need anymore, right?
https://github.com/syncthing/syncthing/blob/master/lib/model/rwfolder.go#L248

Reply:
I guess we can keep that and scrap my print.

Reply:
I actually like your print better, as it preserves temporal order ;)